themes/inbeat/layouts/partials/site/showcase-popup.html

```diff
@@ -0,0 +1,102 @@
+<section
+  id="showcasePopup"
+  :class="['showcase-popup', popupClass]"
+  v-cloak
+  v-if="showPopup">
+  <div class="sc-popup">
+    <div class="content">
+      <div class="close" @click="hideShowcasePopup">
+        {{- partial "icons/close.html" -}}
+      </div>
+      <img src="/images/showcase-img.png" alt="Showcase">
+      <div class="right-content">
+        <span class="new-badge">NEW</span>
+        <h1 class="title">UGC Platform</h1>
+        <p class="text">Create high quality UGC with top creators.</p>
+        <a href="https://www.showca.se/" title="Sign up for free &#8212 On Showcase" class="btn btn-lg" target="_blank">
+          <span>Sign up for free</span> &#8212 On Showcase
+      </a>
+      </div>
+    </div>
+  </div>
+</section>
+
+{{- if getenv "IS_PROD" -}}
+<script src="https://cdn.jsdelivr.net/npm/vue@2.6.12"></script>
+{{- else -}}
+<script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
+{{- end -}}
```

Comment:
Why Vue? I don't mind it but I definitely feel it's not necessary and just gets in the way of our loading time a bit.
Watch out because I think you're loading v2.6.12 in Prod but v3+ in Dev/Staging.

Reply:
Yea, not sure. I just used a template from an other page that use the same logic for quicker integration.
I'll use `https://cdn.jsdelivr.net/npm/vue/dist/vue.js` for prod also!

Reply:
Actually, it's cleaner if you pin a version. I don't mind which version, you can decide but just keep the same for both env :) You can clean it up on the other page too if you feel like it. When it was built, it didn't matter because the main route was leading to Vue2.